src/main/kotlin/no/nav/familie/ks/sak/kjerne/behandling/steg/vilkårsvurdering/regelsett/lov2021/ForskyvVilkår2021.kt

```diff
@@ -1,22 +1,17 @@
 package no.nav.familie.ks.sak.kjerne.behandling.steg.vilkårsvurdering.regelsett.lov2021
 
-import no.nav.familie.ks.sak.common.tidslinje.Periode
-import no.nav.familie.ks.sak.common.util.DATO_LOVENDRING_2024
-import no.nav.familie.ks.sak.common.util.TIDENES_ENDE
-import no.nav.familie.ks.sak.common.util.TIDENES_MORGEN
-import no.nav.familie.ks.sak.common.util.førsteDagIInneværendeMåned
-import no.nav.familie.ks.sak.common.util.sisteDagIMåned
 import no.nav.familie.ks.sak.kjerne.behandling.steg.vilkårsvurdering.domene.Vilkår
 import no.nav.familie.ks.sak.kjerne.behandling.steg.vilkårsvurdering.domene.VilkårResultat
 import no.nav.familie.ks.sak.kjerne.behandling.steg.vilkårsvurdering.regelsett.lov2021.barnehageplass.forskyvBarnehageplassVilkår
-import no.nav.familie.ks.sak.kjerne.behandling.steg.vilkårsvurdering.regelsett.mapTilTilknyttetVilkårResultater
+import no.nav.familie.ks.sak.kjerne.behandling.steg.vilkårsvurdering.regelsett.lov2021.standard.forskyvStandardVilkår
 
 fun forskyvEtterLovgivning2021(
     vilkårType: Vilkår,
     vilkårResultater: List<VilkårResultat>,
 ) = when (vilkårType) {
-    Vilkår.BARNEHAGEPLASS -> {
-        vilkårResultater.forskyvBarnehageplassVilkår()
+    Vilkår.BARNEHAGEPLASS,
+    -> {
+        forskyvBarnehageplassVilkår(vilkårResultater)
     }
```

Comment:
```suggestion
```suggestion
    Vilkår.BARNEHAGEPLASS
    -> forskyvBarnehageplassVilkår(vilkårResultater)
```
```